BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -322,17 +306,18 @@ - (void)connection:(NSURLConnection *)connection willSendRequestForAuthenticatio
 
 		if (shouldTrustServer)
 		{
-            [self completeServerTrustAuthenticationChallenge:challenge shouldTrust:shouldTrustServer];
+            [self completeServerTrustAuthenticationChallenge:challenge];
         }
 		else
 		{
-            UIAlertView *serverTrustAlertView = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"Cannot Verify Server Identity", @"Alert view title: title for SSO server trust challenge")
-                                                                           message:[NSString stringWithFormat:NSLocalizedString(@"Login failed because app cannot verify the identity of \"%@\".", @"Alert view message: Message for SSO server trust challenge, giving the user the host of the server who's identity cannot be verified."), [[challenge protectionSpace] host]]
-                                                                          delegate:self
-                                                                 cancelButtonTitle:NSLocalizedString(@"OK", @"Button title: Dismiss the alert view")
-                                                                 otherButtonTitles:nil];
-            serverTrustAlertView.tag = BOX_SSO_SERVER_TRUST_ALERT_TAG;
-            [serverTrustAlertView show];
+            UIAlertView *loginFailureAlertView = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"Login Failure", @"Alert view title: Title for failed SSO login due to authentication issue")
+                                                                            message:NSLocalizedString(@"Could not complete login because the SSO server is untrusted. Please contact your administrator for more information.", @"Alert view message: message for failed SSO login due to untrusted (for example: self signed) certificate")
```

Comment:
Do you think we should include the host we encountered? I think that could be valuable when the users reach out to their admins.